godtools/Views/TractElements/TractTabs+ChildSetup.swift

```diff
@@ -19,7 +19,7 @@ extension TractTabs {
             for node in item.children {
                 if self.xmlManager.parser.nodeIsLabel(node: node) {
                     if let textNode = self.xmlManager.parser.getTextContentFromElement(node) {
-                        self.tabsProperties().options.append(textNode.text!)
+                        self.tabsProperties().options.append(textNode.text)
```

Comment:
Why was the `!` removed?

Reply:
This was a required as part of building in XCode 9. I'm not sure why, but the build would not pass w/o this change.